avm/src/daemon/dns.rs

```diff
@@ -45,6 +45,17 @@ impl VMMetadata {
       Err(_) => return Err("Data in TXT record is not a valid string".into()),
     }
   }
+
+  fn get_local_metadata() -> DaemonResult<Vec<VMMetadata>> {
+    Ok(vec![VMMetadata {
+      schema_version: "v1".to_string(),
+      alan_version: option_env!("CARGO_PKG_VERSION").unwrap().to_string(),
+      cloud: "LOCAL".to_string(),
+      private_ip_addr: "127.0.0.1".to_string(),
+      region: "localhost".to_string(),
+      public_ip_addr: "127.0.0.1".to_string(),
+    }])
+  }
```

Comment:
Why is this a function when it's used a single time with a hardwired result? Probably better to just inline it where it's called *or* make it a global const OnceCell so it isn't reconstructed on every call to `get_vms` below?

Reply:
Agree, should be static